src/menu.h

```diff
@@ -1,9 +1,6 @@
+#pragma once
+
 #include "globals.h"
 #include "glyphs.h"
 
-#ifndef _MENU_H_
-#define _MENU_H_
-
 void ui_idle(void);
```

Comment:
A bit weird that the header declaring the ui function is not in the ui subdirectory, isn't it ?

Reply:
Moved to UI

Reply:
src/ui/ui_api.h with all UI calls